tests/unit/store/unload-test.js

```diff
@@ -86,7 +86,7 @@ test("unload a record", function(assert) {
         store.unloadRecord(record);
       });
 
-      assert.equal(get(record, 'hasDirtyAttributes'), false, "record is not dirty");
```

Comment:
Using Ember 1.13.13, after a record was unloaded/destroyed, `get(record, 'hasDirtyAttributes')` returned `undefined`. Since the record was now destroyed, I felt as though checking `hasDirtyAttributes` was no longer valid. Replaced with an `isDestroyed` check. Would it be preferred that `hasDirtyAttributes` responds with `false`?